src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -137,14 +137,16 @@ Amplify.configure({
 
 ## Configure frontend
 
+<Callout>
+
+Make sure that the `@aws-amplify/interactions` package has the same version number as the `aws-amplify` package in your `package.json` file.
+
+</Callout>
+
 import js0 from '/src/fragments/lib/interactions/js/frontend.mdx';
 
 <Fragments fragments={{ javascript: js0 }} />
 
-import reactnative1 from '/src/fragments/lib/react-native-polyfills.mdx';
```

Comment:
this fragment is under `/src/fragments/lib/`. Is it only used in `interactions`?

Reply:
![image](https://github.com/aws-amplify/docs/assets/4989523/c0ae3759-3f34-4760-b3be-32c6faced7fa)
yes